### PR TITLE
[rllib] Set PPO observation filter to NoFilter by default

### PR DESCRIPTION
**Pull request by ericl**
## What do these changes do?

This has come up as an common "gotcha" enough times we should flip the default value. Add a warning to avoid silent breakage of existing workloads.

### CODE REVIEW

**Comment by ericl** [comment]
@richardliaw any thoughts? It is a kind of scary breaking change, but the default is pretty bad right now.

File: python/ray/rllib/agents/ppo/ppo.py [hunk 1]
```diff
@@ -36,7 +36,7 @@
     # Share layers for value function
     "vf_share_layers": False,
     # Coefficient of the value function loss
-    "vf_loss_coeff": 1.0,
+    "vf_loss_coeff": 0.01,
```
Comment (ericl) [inline python/ray/rllib/agents/ppo/ppo.py]: While we're breaking hyperparams, make it consistent with IMPALA.

**Comment by richardliaw** [comment]
Hm yeah, the change is probably necessary, but for this, you probably want a very visible section in the docs that highlight the most common issues/possible regressions (and then point to any of the recent breaking changes for people to diagnose).

**Comment by ericl** [comment]
I feel like a note in docs would be easily missed. I changed it to print out each PPO iteration (can be suppressed with log_level or by setting the config)?

File: python/ray/rllib/agents/ppo/ppo.py [hunk 2]
```diff
@@ -99,6 +99,12 @@ def _init(self):
 
     @override(Agent)
     def _train(self):
+        if "observation_filter" not in self.raw_user_config:
+            # TODO(ekl) remove this message after a few releases
+            logger.info(
+                "Important! Since 0.7.0, observation normalization is no "
+                "longer enabled by default. To enable running-mean "
+                "normalization, set 'observation_filter': 'MeanStdFilter'.")
```
Comment (richardliaw) [inline python/ray/rllib/agents/ppo/ppo.py]: Maybe release at most 3 times and then add a note that this message can be ignored if learning performs as expected?

Comment (ericl) [reply python/ray/rllib/agents/ppo/ppo.py]: ok done

**Review by richardliaw — APPROVED** [review]

**Pull request merged**